tests/client.rs

```diff
@@ -1827,7 +1827,7 @@ mod conn {
             sock.set_write_timeout(Some(Duration::from_secs(5)))
                 .unwrap();
             let mut buf = [0; 4096];
-            sock.read(&mut buf).expect("read 1");
+            let _ = sock.read(&mut buf).expect("read 1");
```

Comment:
Why does clippy want this change? Seems like more noise, why can't we just ignore the integer?

Reply:
My Vscode shows that it needs handle the return amount. 

Well, this isn't a very valuable change by comparison, and 'let _' is another way to ignore the return value